nixops/backends/__init__.py

```diff
@@ -3,56 +3,42 @@
 import os
 import re
 import subprocess
-from typing import Dict, Any, List, Optional, Union, Set
+from typing import Mapping, Any, List, Optional, Union, Set, Sequence
 import nixops.util
 import nixops.resources
 import nixops.ssh_util
-import xml.etree.ElementTree as ET
+
+
+class MachineOptions(nixops.resources.ResourceOptions):
+    storeKeysOnMachine: bool
+    targetPort: int
+    alwaysActivate: bool
+    owners: Sequence[str]
+    hasFastConnection: bool
+    keys: Mapping[str, Mapping]
```

Comment:
Does this make the interior Mapping `Any, Any`? Just curious. We don't need to fix the world in this PR.

Reply:
Yes, that's the default.